repo/ci.sh/before-install.pre.inc.sh

```diff
@@ -12,7 +12,7 @@ function sf_run_docker_ci_image() {
     echo_done
 
     echo_do "Running the ${CONTAINER_NAME} container, proxying relevant env vars and mounting ${MOUNT_DIR} folder..."
-    exe docker run -d -it --rm \
+    exe docker run -v /var/run/docker.sock:/var/run/docker.sock -d -it --rm \
```

Comment:
As we talked about: check if you can use `--volume` and move this next to the other `--volume` line.